client/packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -15,29 +15,38 @@ import { TypedTFunction, LocaleKey } from '@common/intl';
 
 interface DataRowProps<T extends RecordWithId> {
   columns: Column<T>[];
-  rows: T[];
   onClick?: (rowData: T) => void;
   rowData: T;
   rowKey: string;
   ExpandContent?: FC<{ rowData: T; isExpanded: boolean }>;
   dense?: boolean;
   rowIndex: number;
   keyboardActivated?: boolean;
-  generateRowTooltip: (row: T) => string;
+  generateRowTooltip?: (row: T) => string;
   localisedText: TypedTFunction<LocaleKey>;
   localisedDate: (date: string | number | Date) => string;
   isAnimated: boolean;
 }
+const Animation: FC<PropsWithChildren<{ isAnimated: boolean }>> = ({
```

Comment:
this is the cause of the input losing focus - it was forcing all the children ( the entire table ) to re-render when the data changed. Because the children were being recreated, the focus isn't retained